### DISCUSSION TITLE
What would cause certain binary sensors to be unknown?

### DISCUSSION THREAD

**Discussion by miguelarios** [zwave-js/zwave-js#3472] [Q&A] [answered]
I have the YRD226 Yale Assure lock and I am trying to obtain when a certain key code is used via home assistant or when it's manually locked or unlocked. I researched that alarm level and alarm type should give me this but I am just seeing them stay as unknown.

**Comment by kpine** [comment, marked as answer]
Use notification events. https://www.home-assistant.io/integrations/zwave_js/#node-events-notification

**Comment by miguelarios** [comment]
You are absolutely right! Thank you!
